CONTRIBUTING.md

```diff
@@ -121,4 +125,8 @@ Examples:
 **IMPORTANT**: By submitting a patch, you agree to license your work
 under the same license as that used by the project.
 
-You're free to modify or redistribute the content. That being said, but why not contribute over here? :) Say if you wanted to have `tldr` pages in `groff` format, why not have a client that uses [pandoc](http://johnmacfarlane.net/pandoc/) and periodically updates straight from this repo?
+You're free to modify or redistribute the content.
+That being said, but why not contribute over here? :)
+Say if you wanted to have `tldr` pages in `groff` format,
```

Comment:
Indeed, I hadn't heard of it. :stuck_out_tongue_closed_eyes: 

Well, 
if that's the intention here. 
I am okay with it too.